### PR TITLE
Added pickup functionality

### PR DESCRIPTION
**Pull request by rikmorskate**

### CODE REVIEW

File: src/Data/PickupData.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Soved\Laravel\Sendcloud\Data;
+
+use Carbon\Carbon;
+
+class PickupData extends RecipientData
+{
+    public Carbon $pickup_from;
+    public Carbon $pickup_until;
```
Comment (sander3) [inline src/Data/PickupData.php:10]: Are these values always set?

Comment (rikmorskate) [reply src/Data/PickupData.php:10]: Currently they should be yes https://docs.sendcloud.sc/api/v2/shipping/#create-a-pickup. Currently and for the immediate future Sendcloud only supports DHL Express for pickups and they require a pickup until date.

File: src/Data/PickupData.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    public int $quantity;
+    public string $telephone;
+    public string $email;
```
Comment (rikmorskate) [inline src/Data/PickupData.php:13]: Email is optional now according to Sendcloud

File: src/Data/PickupData.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        'name',
+        'address',
+        'city',
+        'postal_code',
+        'country',
+        'pickup_from',
+        'pickup_until',
+        'quantity',
+        'telephone',
+        'email',
```
Comment (rikmorskate) [inline src/Data/PickupData.php:30]: Email is optional now according to Sendcloud

**Pull request closed without merging**